backend/src/packages/chats/chat.service.ts

```diff
@@ -83,8 +92,24 @@ class ChatService implements Service {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Service['delete']> {
-    return Promise.resolve(true);
+  public async delete({
+    id,
+    userId,
+  }: {
+    id: number;
+    userId: number;
+  }): Promise<boolean> {
+    const allChats = await this.findAllByUserId(userId);
+
+    const chat = allChats.items.find((chat) => {
+      return chat.id === Number(id);
```

Comment:
```suggestion
      return chat.id === id;
```
You already receive it as number

Reply:
Actually, it is a string as it is a param. But I'll move this logic to the controller. Thanks for noticing.